src/ops/ops_test.go

```diff
@@ -313,3 +314,25 @@ var _ = Describe("RemoveAllDMDevicesOnDisk", func() {
 	})
 
 })
+var _ = Describe("Set Boot Order", func() {
+	var (
+		l        = logrus.New()
+		ctrl     *gomock.Controller
+		execMock *execute.MockExecute
+		conf     *config.Config
+	)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		execMock = execute.NewMockExecute(ctrl)
+		conf = &config.Config{}
+	})
+
+	It("Set boot order", func() {
```

Comment:
from what I'm seeing tests are passing even without your changes in ``ops.go``
what am I missing?

Reply:
I have tested on Power if remove my changes, the test will fail, it is platform dependent. Also it passed on amd system.